docs/satellite-clone.md

```diff
@@ -91,3 +91,24 @@ On the target server:
       # ansible-playbook satellite-clone-playbook.yml
     ```
   The playbook run may may take a while to complete.
+
+## Cloning a Satellite With Remote Databases
+
+Note: The backup used for these procedures must have the correct options to manage remote databases in `/etc/foreman-installer/scenarios.d/satellite-answers.yaml`. Please check that the answers file has the correct database management values filled in the configuration backup file. These will be values such as `db-manage` in the `foreman` section and `pulp-manage-db` in the `katello` section. They are currently (at the time of this writing) [missing from the initial setup documentation](https://bugzilla.redhat.com/show_bug.cgi?id=1887846)
```

Comment:
I think this puts a large burden on the user to work around our own bug. Couldn't we handle this during the playbook run?